### PR TITLE
Print config

### PR DESCRIPTION
**Pull request by angainor**

### CODE REVIEW

File: benchmarks/args.hpp [hunk 1]
```diff
@@ -27,7 +29,10 @@ struct args
 
     args(int argc, char** argv)
     {
-        if (argc != 4) { is_valid = false; }
+        if (argc != 4) {
+            is_valid = false;
+            if(argc==2 && !std::strcmp(argv[1], "-c")) oomph::print_config();
```
Comment (boeschf) [inline benchmarks/args.hpp:34]:
```suggestion
            if(argc==2 && !std::strcmp(argv[1], "-c")) print_config();
```

File: benchmarks/args.hpp [hunk 2]
```diff
@@ -27,7 +29,10 @@ struct args
...
+        }
```
Comment (boeschf) [inline benchmarks/args.hpp:35]: please clang-format it

File: include/oomph/utils.hpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#pragma once
+
+namespace oomph
+{
+  void print_config();
+}
```
Comment (boeschf) [inline include/oomph/utils.hpp:15]: pls clang-format

File: src/utils.cpp [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+void print_config()
+{
+    std::cout << std::endl;
+    std::cout << "compile configuration:" << std::endl;
+    std::cout << std::endl;
+#include <oomph/cmake_config.inc>
+    std::cout << std::endl;
+}
+
+}
```
Comment (boeschf) [inline src/utils.cpp:24]: pls clang-format

**Comment by angainor** [comment]
@boeschf hopefully I `clang-format` correctly.

**Pull request merged**